README.md

```diff
@@ -87,10 +87,9 @@ The available options are:
   -b, --beautify                Beautify output/specify output options.
   -m, --mangle                  Mangle names/pass mangler options.
   -r, --reserved                Reserved names to exclude from mangling.
-  -c, --compress                Enable compressor/pass compressor options. Pass
-                                options like -c
-                                hoist_vars=false,if_return=false. Use -c with
-                                no argument to use the default compression
+  -c, --compress                Enable compressor/pass compressor options, e.g.
+                                `-c if_return=false,pure_funcs=["Math.pow","console.log"]`
```

Comment:
That might work in the Windows shell, but in the bash shell on UNIX you have to do this:
```
-c 'if_return=false,pure_funcs=["Math.pow","console.log"]'
```
I'm not sure there's a platform independent way to document `pure_funcs`.